### ISSUE TITLE
Add external layer does not generate summary.csv file

### ISSUE THREAD

**Issue by SaraMederos** [brainglobe/brainreg-segment#96]
**Describe the bug**
Implemented add region from selected layer successfully generates the volumes, it fails to generate the summary file
**To Reproduce**
Generate a layer by using a threshold over an image (i.e. Otsu) - run add region from selected layer -- run analysis
**Expected behaviour**
Generate .cdv volumes and summary files

**Log file**
[logErr.txt](https://github.com/brainglobe/brainreg-segment/files/11589144/logErr.txt)

**Comment by adamltyson** [comment]
I can reproduce this and it seems to occur due to some plugins creating napari layers with underlying data types not supported by numpy. Created #99 to fix this.

**Comment by adamltyson** [comment]
Hi @SaraMederos could you try `pip install brainreg-segment==0.2.17rc3` to see if this resolves the issue?

As an aside, once you're happy this all works, we should include your tutorial using this new feature into the new docs website (currently at https://brainglobe.github.io/new-website/).

**Comment by SaraMederos** [comment]
I have tried the new version, and it performs well and runs perfectly in the sample space. However, I encountered a different error when attempting the same process in the atlas space. If that's the case, and we want to debug that specific issue, I will open a new ticket for it. In the meantime, I will make some changes to the documentation version I created, and I will upload it by this Friday so that it can be added to the new webpage.
